Microsoft.NET.Build.Containers/Image.cs

```diff
@@ -7,6 +8,14 @@ namespace Microsoft.NET.Build.Containers;
 
 record Label(string name, string value);
 
+public enum PortType
+{
+    tcp,
+    udp
```

Comment:
.NET style guide says these should start with a capital letter, but I do not feel super strongly about it.

Reply:
Definitely doable, I'd just worry about the parsing side - the spec is case-sensitive for these values.

Reply:
added comments as to _why_ they are lowercased.